README.md

```diff
@@ -730,7 +730,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Utkarsh Tomar](https://github.com/Utkarsh736)
```

Comment:
```suggestion
  - [Utkarsh Pathak](https://github.com/utkarsh0308)
  - [Utkarsh Tomar](https://github.com/Utkarsh736)
```

Reply:
Hey, @Panquesito7  **` Utkarsh Pathak`** should be added below  **` Utkarsh Mishra`**

Because the last name come in order (M, P, T)
1. Mishra
2. Pathak
3. Tomar

Reply:
@Pradumnasaraf I have done as u suggested plzz check once...

Reply:
> @Pradumnasaraf I have done as u suggested plzz check once...

Not changes are Okay

Reply:
I think it is good as-is? According to an alphabetical order page, it is good.

Reply:
?

Reply:
> I think it is good as-is? According to an alphabetical order page, it is good.

?